Src/BlazorApp/Pages/Races.razor

```diff
@@ -13,128 +13,149 @@
 <input @bind="_name"/>
 
 <div class="container">
-    <table class="table" style="margin-top: 10px;">
-        <thead>
-        <!-- Should be hided if there is no races -->
-        <tr>
-            <th>Race id</th>
-            <th>Race name</th>
-            <th>Start time</th>
-        </tr>
-        </thead>
-        <tbody>
-            @{
-                foreach (var race in _races)
+    <div class="row">
+
```

Comment:
@terjetopland refactored to have races and participants side by side.

Reply:
At the moment we don't get any result even if we have set endtime on a participant, but suppose this is a quick fix...